utils/src/api_schema/api_schema.json

```diff
@@ -663,13 +663,14 @@
 
     "AddVideo": {
       "properties": {
-        "_ref":         { "$ref": "#/definitions/refInt" },
-        "index_frames": { "type": "boolean" },
-        "codec":        { "$ref": "#/definitions/vidCodecString" },
-        "container":    { "$ref": "#/definitions/vidContainerString" },
-        "link":         { "$ref": "#/definitions/blockLink" },
-        "operations":   { "$ref": "#/definitions/blockVideoOperations" },
-        "properties":   { "type": "object" }
+        "_ref":             { "$ref": "#/definitions/refInt" },
+        "index_frames":     { "type": "boolean" },
+        "from_local_file" : { "type": "string"},
```

Comment:
"from_local_file", from the user perspective, seems like it is a local file in the client machine. Maybe the right name should be "from_server_file" or something of the kind. what do you think?

Reply:
I agree, I'll change it to 'from_server_file'.